__tests__/mockClient.test.js

```diff
@@ -1,11 +1,15 @@
 const Q = require('q');
 const request = Q.denodeify(require('request'));
 const MOCKSERVER_URI = 'localhost:3000';
+const mockServer = require('../src/server/index.js');
 const MockClient = require('../src/client/Client')(MOCKSERVER_URI);
 const MOCKED_HOST_URI = 'http://localhost:4000';
 const mockClient = new MockClient(4000);
 
 describe('when adding mocks on the same uri', () => {
+	beforeAll(() => mockServer.start());
```

Comment:
We are using Jest (similar to Jasmine). It supports promises, so it will automatically wait for the promise to resolve.